src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBScriptBindingProvider.cs

```diff
@@ -74,7 +75,14 @@ public override Type DefaultType
                 {
                     if (Context.Access == FileAccess.Read)
                     {
-                        return typeof(JObject);
+                        if (string.Compare("many", Context.Cardinality, StringComparison.OrdinalIgnoreCase) == 0)
+                        {
+                            return typeof(IEnumerable<JObject>);
```

Comment:
Is this the place that should be JArray instead? We don't want to require a corresponding script  change. 

---
In reply to: [102275264](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/182#discussion_r102275264) [](ancestors = 102275264)